content/ruby/concepts/argf/argf.md

```diff
@@ -0,0 +1,130 @@
+---
+Title: 'ARGF and ARGV'
+Description: 'ARGF is a module in Ruby used for reading files passed as command line arguments or in standard input (STDIN) in a terminal. A Ruby script using ARGF may be thought of as implementing a subset of the functions of the Unix command `cat`.'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'ARGF'
+  - 'ARGV'
+  - 'Standard Input'
+CatalogContent:
```

Comment:
While we have two entries for CatalogContent, generally one is to one of the paid course paths, an appropriate one is `paths/front-end-engineer-career-path`